internal/namespaces/instance/v1/custom_server_create.go

```diff
@@ -255,8 +255,8 @@ func instanceServerCreateRun(ctx context.Context, argsI interface{}) (i interfac
 	if len(args.AdditionalVolumes) > 0 || args.RootVolume != "" {
 		// Get default organization ID.
 		organizationID := args.OrganizationID
-		if organizationID == "" {
-			organizationID = core.GetOrganizationIDFromContext(ctx)
+		if organizationID == nil {
+			organizationID = scw.StringPtr(core.GetOrganizationIDFromContext(ctx))
```

Comment:
⚠️ if core.GetOrganizationIDFromContext(ctx) return `""` it should be configured as nil